src/HttpTracer/HttpTracerHandler.cs

```diff
@@ -30,10 +30,11 @@ public HttpMessageParts Verbosity
         /// </summary>
         /// <param name="handler">User defined <see cref="HttpMessageHandler"/></param>
         /// <param name="logger">User defined <see cref="ILogger"/></param>
-        public HttpTracerHandler(HttpMessageHandler handler = null, ILogger logger = null)
```

Comment:
I would go even further than this, to having multiple constructors with "Required" parameters.

Reply:
Might add a fair bit more code, so I will understand if you want to leave the optional like this.

I personally don't like having to create an instance of an object passing null in the constructor =P